felt/src/lib.rs

```diff
@@ -167,6 +167,18 @@ mod test {
             prop_assert!(&x.to_biguint() < p);
         }
 
+        #[test]
+        // Property-based test that ensures, for 100 {x} and {y} values that are randomly generated each time tests are run, that a substraction between two felts {x} and {y} and doesn't fall outside the range [0, p]. The values of {x} and {y} can be either [0] or a very large number.
```

Comment:
The range should be open on the prime side:
```suggestion
        // Property-based test that ensures, for 100 {x} and {y} values that are randomly generated each time tests are run, that a substraction between two felts {x} and {y} and doesn't fall outside the range [0, p). The values of {x} and {y} can be either [0] or a very large number.
```